src/continents.ts

```diff
@@ -0,0 +1,98 @@
+'use strict'
+
+import * as _ from 'underscore'
+import { regions } from './regions.js'
+
+export interface Continent {
+    name: string
+    regions: string[]
+    countries: []
+}
+
+export interface Continents {
+    asia: Continent | {}
```

Comment:
remove ` | {}` everywhere

and then just cast below e.g. `asia: {} as any,`

Reply:
yes sir